motion/thruster_interface_asv/src/thruster_interface_asv_driver_lib.cpp

```diff
@@ -0,0 +1,235 @@
+#include <thruster_interface_asv/thruster_interface_asv_driver_lib.hpp>
+
+namespace thruster_interface_asv_driver_lib {
+// Interpolation from forces to pwm functions (START) ----------
+struct _ForcePWM {
+  float force;
+  float pwm;
+};
+
+// Local variables used inside the interpolation
+std::vector<_ForcePWM> _datatableForForceAndPWM;
+
+std::vector<_ForcePWM> _loadDataFromCSV(const std::string &filepath) {
+  // Prepare the datastructure we will load our data in
+  std::vector<_ForcePWM> data;
+
+  // Try opening the file and if suceeded fill our datastructure with all the
+  // values in the .CSV file
+  std::ifstream file(filepath);
+  if (file.is_open()) {
+    // Variable for the currect line value in the .CSV file
+    std::string line;
+
+    // Skip the header line
+    std::getline(file, line);
+
+    // Read data line by line
+    while (std::getline(file, line)) {
+      // Temporarty variables to store data correctly ----------
+      // Define temporary placeholders for variables we are extracting
+      std::string tempVar;
+      // Define data structure format we want our .CSV vlaues to be ordered in
+      _ForcePWM formatOfForcePWMDatastructure;
```

Comment:
maybe use a shorter and more readable variable name, something along the lines of "ForcePWMData".